### PR TITLE
Add PayPay

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: img/p/paypay.ne.jp.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...x00jvnh91vmRcLAAAflUxTTM1TtEdcyKxGs6QpbxPqM6vxFqWoTO8ZORXcjq26pqnb4bMZVOszL6A2fjeq4tux/LER9GseUwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABMH0btM7vxDqWpV0704uPFqmfvVz/AEpn3pONTrVMtE37yuhjW8eP1Tr8o/eVhkxy0AABzXMrUvBPAutZcTEVRj1W6J+9X4kfGpbu1dGiZZfYON61tGza+OvyjjPkp8xzuwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACzH0fNM7z4FnLqjavOyK7m/3afEj4xPvTsenSnXtci31yvS7Q9FHKiIj5zx+8JOX2ngAAIi+kfqfe/DOBp9NURVl5HTmPPTRH9aqUbJn2YhvO4uN08uu/P6Y0+c/tEq7IbqgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABETMxERvPkFF0OEdNjR+GNL0/barHx6KKuvfxtvG+O7J0U9GmIcB2nk+t5d2//NVM/Lq+jbPSCAAArX9IfU+++NLOFTVvRhY9NMxE9ldfjT8Oig5E616djrW5GL6LAm7POuqfCOHnqi1YbmAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA6Ll3pnhfjfRsOY3oryaa64+7T41Xwpl7t09KqIYrbeT6rs+9d7KZ0754R5riMk4OAAAT1QCmnG+p+GOLtXz9+lTeya5onffxYnan4RDG1z0qpl3zZON6phWrPZTHjzn6tI8MiAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkj6P8RPMK1MxEzGNdmPR1QvWP4jUt9J/wDtk/8AlSs8nuPgAAMbU5mnTcuaZmJi1XMTHk8WVJ5L2PGt2mJ7Y81ImLh9DgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAP/Z"/></g></svg>
```
Comment (Carlgo11) [inline img/p/paypay.ne.jp.svg]: This SVG contains a JPEG image. Please replace it with a purely SVG image.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by ghost** [comment]
> I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

There is no documentation, but there is some mention of it on the help page.
https://paypay.ne.jp/help/c0012/

File: img/p/paypay.ne.jp.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....72,19.55,0,14.26,10.82,18.91,18.5,18.91A23.34,23.34,0,0,0,270,324.6v2.92h15.7V288.57c0-12.31-12.25-16.46-24.15-16.46m.67,49.43c-6.83,0-11.73-5-11.73-12.46S257,297.26,263,297.26c3,0,5.87.83,7,1.78V318.1c-1.93,1.67-4.17,3.44-7.78,3.44m60.2,12.71c-4.77,10.55-12.64,11.93-16.14,12.33a54.91,54.91,0,0,1-7.65.25V338.2c7,0,10.82.55,14.95-7.64l2.8-7.1-25.81-51.35h16.49L324,306.25l15-34.14h10.71s-22.46,51.59-27.23,62.14M26,253.52H3.69A3.69,3.69,0,0,0,0,257.21v70.31H15.72V302.37H26c15.22,0,29-9.49,29-25.57s-13.79-23.28-29-23.28M15.72,294.8V261.08s4.85,0,10.29,0c7.71,0,16.22,5.86,16.22,15.89S33.72,294.81,26,294.81H15.72m66.07-22.69c-11.9,0-18,2.93-18,2.93l1.46,4.41s4.48-2.08,13.25-2.08c9.76,0,11.75,4.74,11.75,11.2l-.14,2.89a49.61,49.61,0,0,0-12-1.42c-7.44,0-19.72,4.59-19.72,19.55,0,14.26,10.82,18.91,18.5,18.91a23.4,23.4,0,0,0,13.33-3.9v2.92h15.7V288.57c0-12.31-12.25-16.46-24.15-16.46m.66,49.43c-6.83,0-11.72-5-11.72-12.46s6.54-11.82,12.52-11.82c3,0,5.87.83,7,1.78V318.1c-1.93,1.67-4.17,3.44-7.79,3.44m60.21,12.71c-4.77,10.55-12.64,11.93-16.15,12.33a54.78,54.78,0,0,1-7.64.25V338.2c7,0,10.82.55,14.94-7.64l2.8-7.1L110.8,272.11h16.49l16.92,34.14,15-34.14h10.7s-22.45,51.59-27.22,62.14"/></g></g></svg>
```
Comment (Carlgo11) [inline img/p/paypay.ne.jp.svg]: Cropped and minified version:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 220 220"><rect width="220" height="220" fill="#f03" ry="31"/><path fill="#fff" d="M11.9 30.5c-4 0-8 0-12 .2v38.1c26.7-2.8 53.6-3 80.4-.5L42.7 220H56l10-41.5c67-5.1 108.2-20.7 117.1-61.2 9.6-43.7-49.4-87.5-171.2-86.8Zm80.3 39c54.3 7.4 88.2 27 83 48-5.5 22.6-57.2 36.3-102.8 34.2z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**